### ISSUE TITLE
testing courtcheck

### ISSUE THREAD

**Issue by ta946** [AggieSportsAnalytics/CourtCheck#21]
are you willing to share the trained model or the videos used for the annotation so that its possible to test courtcheck?

**Comment by corypham** [comment]
Yes absolutely, the data I used to train the model should be in the google drive and the models as well. Below is the link to both the court and ball detection model weights:

https://drive.google.com/drive/folders/1-JgHon0KhACLNjbEjew3bHDN86QvoL01?usp=drive_link
